src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -87,6 +87,7 @@ export interface IOncoprintControlsState
     onlyShowClinicalLegendForAlteredCases?: boolean;
     showOqlInLabels?: boolean;
     showMinimap: boolean;
+    isSaveButtonEnabled: boolean;
```

Comment:
is this too general a name?

Reply:
Fixed in [9065f7e](https://github.com/cBioPortal/cbioportal-frontend/compare/240ae85eba446230bffd086012e309356e71f413..9065f7ea67dda257aad80c7a2b09cc88272492d9)